modules/theme/src/themes/default/collections/table.overrides

```diff
@@ -85,6 +85,20 @@
                     }
                 }
             }
+
+            &.actions-cell {
+                &.items-1 {
+                    min-width: 47px;
```

Comment:
Shall we take these from variables

Reply:
Resolved by ac7d554dc4f5661cc86710176a69bbeb17e7f021